### DISCUSSION TITLE
In an 'options' list for a 'select' filter, how to add a wildcard?

### DISCUSSION THREAD

**Discussion by ottadini** [rclement/datasette-dashboards#202] [Q&A] [answered]
I have a column with, say, three text values [past, present, future]. In the options list for the select filter, how can I specify a wildcard to allow choice of all options? I tried *, but this threw a syntax error.

**Comment by HaveF** [comment, marked as answer]
You can leave it as blank

**Reply by rclement** [reply]
Thanks @HaveF for answering this question 👍 

Here are some more context, see how it works in the live demo:
- Metadata specifying a select filter: https://github.com/rclement/datasette-dashboards/blob/d879ea348567519f62b9e79b01aa441185e584f4/demo/metadata.yml#L45C16-L45C16
- In a dashboard, a blank option is automatically rendered in the dropdown: https://datasette-dashboards-demo.vercel.app/-/dashboards/job-offers-stats?date_start=2021-01-01&date_end=&source=&region=&company=&job_title=
